.github/workflows/hvd-tests.yml

```diff
@@ -29,6 +29,28 @@ jobs:
       with:
         miniconda-version: "latest"
         python-version: ${{ matrix.python-version }}
+        use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+
+    - name: Get year & week number
+      id: get-date
+      run: |
+        echo "::set-output name=date::$(/bin/date "+%Y-%U")"
+      shell: bash
+
+    - name: Get pip cache dir
+      id: pip-cache
+      run: |
+        conda install pip
+        echo "::set-output name=dir::$(pip cache dir)"
```

Comment:
seems like it does not work : https://github.com/pytorch/ignite/pull/1455/checks?check_run_id=1385363291#step:5:30

Reply:
I think this is not a good sign : https://github.com/pytorch/ignite/pull/1455/checks#step:5:15 :)
Probably, `shell: bash -l {0}` is missing

Reply:
because of pip? not pip3? I think

Reply:
I think if we do `which pip` it will point out to the system's one and not miniconda's one. And we would like to cache miniconda's pip right ?

Reply:
pip3 works

Reply:
Yeah, it works and seems like we are not even required to install pip, right ?

Reply:
I am not quite sure about not upgrading pip, so i would include it